docs/src/userguide/special_methods.rst

```diff
@@ -71,11 +71,22 @@ your :meth:`__cinit__`` method to take no arguments (other than self) it
 will simply ignore any extra arguments passed to the constructor without
 complaining about the signature mismatch.
 
-.. Note: Older Cython files may use :meth:`__new__` rather than :meth:`__cinit__`. The two are synonyms.
-  The name change from :meth:`__new__` to :meth:`__cinit__` was to avoid
-  confusion with Python :meth:`__new__` (which is an entirely different
-  concept) and eventually the use of :meth:`__new__` in Cython will be
-  disallowed to pave the way for supporting Python-style :meth:`__new__`
+.. Note::
+
+    Older Cython files may use :meth:`__new__` rather than :meth:`__cinit__`. The two are synonyms.
+    The name change from :meth:`__new__` to :meth:`__cinit__` was to avoid
+    confusion with Python :meth:`__new__` (which is an entirely different
+    concept) and eventually the use of :meth:`__new__` in Cython will be
+    disallowed to pave the way for supporting Python-style :meth:`__new__`
+
+..  Note::
```

Comment:
I'd move the second note above the first, as it's the more important one. The legacy code reference is no longer relevant for most users.